test/test_browser.py

```diff
@@ -3362,6 +3373,15 @@ def test_async_mainloop(self):
       print(opts)
       self.btest_exit('browser/async_mainloop.cpp', args=['-O' + str(opts), '-sASYNCIFY'])
 
+  @no_firefox('no JSPI support yet')
+  def test_async_pthread_join(self):
```

Comment:
To these tests actually make `pthread_join` async somehow?

Reply:
No, just that pthread_join works when async pthreads are used. I'll add comment.

Reply:
Might be worth renaming the function, since it ready like the join itself is async (which would also be really cool BTW!)